packages/core/index.js

```diff
@@ -149,44 +149,45 @@ const middy = (handler) => {
   const errorMiddlewares = []
 
   const instance = (event, context, callback) => {
-    instance.event = event
-    instance.context = context
-    instance.callback = callback
-    instance.response = null
-    instance.error = null
+    const request = {}
+    request.event = event
+    request.context = context
+    request.callback = callback
+    request.response = null
+    request.error = null
 
     const middyPromise = new Promise((resolve, reject) => {
       const terminate = (err) => {
         if (err) {
           return callback ? callback(err) : reject(err)
         }
 
-        return callback ? callback(null, instance.response) : resolve(instance.response)
+        return callback ? callback(null, request.response) : resolve(request.response)
       }
 
       const errorHandler = err => {
-        instance.error = err
-        return runErrorMiddlewares(errorMiddlewares, instance, terminate)
+        request.error = err
+        return runErrorMiddlewares(errorMiddlewares, request, terminate)
       }
 
-      runMiddlewares(beforeMiddlewares, instance, (err) => {
+      runMiddlewares(beforeMiddlewares, request, (err) => {
         if (err) return errorHandler(err)
 
         const onHandlerError = once((err) => {
-          instance.response = null
+          request.response = null
           errorHandler(err)
         })
 
         const onHandlerSuccess = once((response) => {
-          instance.response = response
-          runMiddlewares(afterMiddlewares, instance, (err) => {
+          request.response = response
+          runMiddlewares(afterMiddlewares, request, (err) => {
             if (err) return errorHandler(err)
 
             terminate()
           })
         })
 
-        const handlerReturnValue = handler.call(instance, instance.event, context, (err, response) => {
+        const handlerReturnValue = handler.call(request, request.event, context, (err, response) => {
```

Comment:
can you change `context` to `request.context`. There was another PR that modified this line.